easybuild/easyconfigs/d/dftd3-lib/dftd3-lib-0.10_fix-extras-syntax.patch

```diff
@@ -0,0 +1,21 @@
+diff -Nru dftd3-lib-0.10.orig/prg/extras.f90 dftd3-lib-0.10/prg/extras.f90
```

Comment:
Sorry, should have asked straight away, but: can you include some comment in the patch about which issue it solves? (e.g. without this patch, you get error XYZ). If there is an official issue or PR on the upstream github of dftd3-lib, it's also good to reference that here. That way, we can easily check when creating newer versions of this EasyConfigs if the patch is still relevant/needed, or if a solution has already been implemented upstream

Reply:
E.g. see https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/PyTorch/PyTorch-1.10.0_fix-alias-violation-in-bitwise-ops.patch